dart/lib/src/sentry_client.dart

```diff
@@ -87,7 +87,6 @@ class SentryClient {
   }) {
     final event = SentryEvent(
       exception: exception,
-      stackTrace: stackTrace,
```

Comment:
this is the stackTrace given by the user, so we'd need to find a way to still pass this along to be used on `SentryExceptionFactory` and `SentryStackTraceFactory`

Reply:
✅